src/handbook/product/strategy.md

```diff
@@ -4,93 +4,67 @@ navTitle: Strategy
+**Our strategic goal is to be the leading "Low-Code IIoT Application Platform"**
 
-### Node-RED Deployments at the Edge (Device Management)
+An IIoT platform is a comprehensive solution that securely collects, integrates, and manages vast volumes of data from IoT endpoints. It's the bridge between Information Technology (IT) and Operational Technology (OT), bringing about a seamless fusion of diverse protocols and data formats, facilitating edge and cloud analytics, and enhancing overall asset management. This digital conduit not only augments traditional OT functionalities but also propels sectors like manufacturing and energy towards a more digitized, efficient future. A robust IIoT platform encapsulates essential elements such as unwavering security, sophisticated automation, and a commitment to sustainability. It fosters an environment where interoperability with a myriad of business applications is not just a feature but a foundational aspect. From meticulous device management and seamless integration to advanced data analytics and application development (see also [IIoT platform definition](https://www.gartner.com/doc/reprints?id=1-2C757S9J&ct=230105&st=sb)).
 
-Many organizations position Node-RED instances on remote servers, edge or industrial devices, to meet network, infrastructure, or use case requirements. Therefore, management of remote instances is crucial for the overall success of closing the gap between IT and OT. 
+Click [here](./verticals.md#industrial-iot-platforms) for more details about the "Low-Code IIoT Application Platforms" market.
 
-FlowFuse already supports this process by creating snapshots on Node-RED instances that can be deployed to multiple remote targets. We will continue to expand the functionalities to allow users to manage existing Node-RED instances, improve their development practices to accelerate development and ensure a reliable and repeatable process, providing more flexible remote deployment options and offering the best support for devices in closed and segmented networks as we are facing it in production environments.
+### Market Demand & Positioning
 
-### Enhanced Integration Capabilities
+With its prowess in connecting and processing data from IIoT devices, Node-RED and FlowFuse find a fitting place in the sphere of IIoT platforms, becoming an integral tool in industries like manufacturing where the utility of such data is paramount for fostering operational efficacy and stimulating innovation. We exist to empower bottom-up innovation. We allow companies to professionalize their workflows to business critical applications.
```

Comment:
It seems like what you're saying is; people need to start automating with Node-RED. Their first step in adoption is just a workflow. (Another way to describe data-capture and transformation, I like it)

From that moment onwards they will understand they need to elevate their workflows to applications. Which is where we come in.

No need to mention the features for Node-RED yet, it's clear from the problem statement in the vision.